trigger/patch.go

```diff
@@ -203,9 +203,16 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 		PastTenseStatus:   t.data.Status,
 		apiModel:          &api,
 		githubState:       message.GithubStatePending,
-		githubContext:     "evergreen",
 		githubDescription: "tasks are running",
 	}
+
+	if t.patch.IsChild() {
+		lastFourPatchID := t.patch.Id.Hex()[len(t.patch.Id.Hex())-4:]
+		data.githubContext = fmt.Sprintf("evergreen/%s/%s", t.patch.Project, lastFourPatchID)
```

Comment:
I think for new projects the Project field will not be intelligible. Instead, we'd want the project's Identifier.
We could potentially fetch the projectRef with its ID here.